### PR TITLE
test: add test for header_toggler.js

### PR DESCRIPTION
**Pull request by dulerong**
## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Other (please describe):

## What is the current behaviour?

Issue Number(s): N/A
No test for header_toggler.js

## What is the new behaviour?

- Add test for header_toggler.js

## Other information

N/A

### CODE REVIEW

**Review by ilunglee — APPROVED** [review]
Looks good to me. Thanks, @dulerong!

File: app/javascript/adminterface/lib/header_toggler.js [hunk 1]
```diff
@@ -38,17 +38,19 @@ class HeaderToggler {
...
-      } else {
-        _self._add()
-      }
-    })
+    this.element.addEventListener('click', (e) => this._clickCallBack(e, $container, _self))
 
     adminterface.addObserver(this.element, this, this.constructor.name)
   }
+
+  _clickCallBack (e, $container, _self) {
```
Comment (ilunglee) [inline app/javascript/adminterface/lib/header_toggler.js:46]: Typo should be `_clickCallback`

**Review by ilunglee — CHANGES_REQUESTED** [review]
Actually, I just find a minor typo. Thanks!

File: app/javascript/adminterface/lib/header_toggler.js (hunk 1 above)
Comment (dulerong) [reply app/javascript/adminterface/lib/header_toggler.js:46]: @ilunglee 
Sorry, so should I just edit the file directly on Github? Or should I edit it in my local machine, rebase, then force push to github? Like, what what you do in this situation?

Comment (ilunglee) [reply app/javascript/adminterface/lib/header_toggler.js:46]: @dulerong,
You can make the changes to the branch on your local machine. And after you finished updating, you can: 
```
# stage the changes
$ git add .

# combine the changes with the latest commit in the history without editing the message
$ git commit --amend --no-edit
```
Then, you can force push to the remote`origin/jest_test_header_toggler` branch.

Cheers,

**Review by ilunglee — APPROVED** [review]

**Pull request merged**